packages/jumpstarter-driver-flashers/oci_bundles/buildroot/replace_kernel.sh

```
@@ -0,0 +1,69 @@
+	mmc_block
+	# R-Car S4 networking
+	gpio-rcar
+	r8a779f0-ether-serdes
+	marvell10g
+	rswitch
+)
+
+[ $? -ne 0 ] && { echo "$0 [target_overlay_dir]"; exit 1; }
+ODIR=$1
+mkdir -p /kernel-automotive
+pushd /kernel-automotive
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)